grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -743,16 +743,26 @@ def section_streaming(outer_panels):
                     ],
                 ),
                 panels.timeseries_count(
-                    "Kafka Consumer Lag Size",
-                    "Kafka Consumer Lag Size by source_id, partition and actor_id",
+                    "Kafka high watermark and source latest message",
+                    "Kafka high watermark by source and partition and source latest message by partition, source and actor",
                     [
                         panels.target(
                             f"{metric('source_kafka_high_watermark')}",
-                            "source={{source_id}} partition={{partition}}",
+                            "high watermark: source={{source_id}} partition={{partition}}",
                         ),
                         panels.target(
                             f"{metric('source_latest_message_id')}",
-                            "source={{source_id}} partition={{partition}} actor_id={{actor_id}}",
+                            "latest msg: source={{source_id}} partition={{partition}} actor_id={{actor_id}}",
```

Comment:
I think it is better to move these two metrics to other section and only keep the Consumer Lag metric under streaming avoid unnecessarily loading detailed info

Reply:
Any suggestion to which section?